package.json

```diff
@@ -7,7 +7,7 @@
     "start": "node .",
     "build": "NODE_ENV=production webpack --config ./config/webpack.prod.js",
     "lint": "npx eslint ./src/**.js --fix",
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
Please, why is a coverage flag added to this script here again? There's a script that collects coverage already

Reply:
I don't think we'll need two separate scripts for test and coverage. What he can do is leave the test script as it is and take out the test:coverage. If that's fine.

Reply:
I have taken out the test:coverage script. Thank you for your observations.